vendor/Makefile

```diff
@@ -131,3 +130,4 @@ $(ALTO2PAGE_ZIP):
 
 $(ALTO2PAGE_DIR): $(ALTO2PAGE_ZIP)
 	$(UNZIP) "$<"
+	mv "JPageConverter 1.5" "$@"
```

Comment:
Okay, but now we have to update the version number here also every time. How about something like
```bash
mv JPageConverter* "$@"
```

Reply:
That could lead to other problems. I'll add a variable.